src/hooks/useBalanceChartData/useBalanceChartData.ts

```diff
@@ -356,7 +353,10 @@ export const useBalanceChartData: UseBalanceChartData = args => {
   const [balanceChartData, setBalanceChartData] = useState<BalanceChartData>(makeBalanceChartData())
 
   const balances = useAppSelector(state =>
-    selectBalanceChartCryptoBalancesByAccountIdAboveThreshold(state, accountId),
+    selectBalanceChartCryptoBalancesByAccountIdAboveThreshold(state, {
+      accountId: accountId ?? '',
+      accountAddress: '', // TODO(gomes)
```

Comment:
Is this something you meant to fix in this PR/

Reply:
Nice catch, we actually shouldn't pass an `accountAddress` in this case, to use the aggregation, that's a left-over from my initial spike on this and will *not* produce the same results if an `accountAddress` (account granularity)

Reply:
8daafb6224770d2108e5d52eb5f652801aff3e45